frontend/components/ResourceCounts.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { Badge, Button, Stack } from '@mantine/core';
+import Link from 'next/link';
+
+/**
+ * Component which retrieves all resources and their counts and translates them into buttons
+ * @returns array of JSX Buttons
+ */
+const ResourceCounts = () => {
+  // set initial state to no resources
+  const [resources, setResources] = useState<Record<string, number>>({
+    Measure: 0,
+    Library: 0
+  });
+  useEffect(() => {
+    // Note: count may not update without refresh if there ends up being a way to change it on the server through the app
+    Promise.all([
+      fetch(`${process.env.NEXT_PUBLIC_MRS_SERVER}/Measure`),
+      fetch(`${process.env.NEXT_PUBLIC_MRS_SERVER}/Library`)
+    ])
+      .then(([resMeasure, resLibrary]) => Promise.all([resMeasure.json(), resLibrary.json()]))
+      .then(([measureBundle, libraryBundle]) => {
+        setResources({ Measure: measureBundle.total || 0, Library: libraryBundle.total || 0 });
+      })
+      .catch(error => {
+        console.log(error.message, '...start the server');
+      });
+  }, []);
+
+  /**
+   * Returns the resource counts key:value pairs object as an array of buttons
+   * @returns array of JSX Buttons that are resources and their counts
+   */
+  const ResourceButtonsGroup = () => {
+    const buttonArray = Object.keys(resources).map(resourceType => (
+      <Link href={`/${resourceType}`} key={resourceType}>
+        <Button
```

Comment:
Similar to my other comment, but I think if we made the button the width of the resources nav bar and also put some white space between the button on the left side it might look better? This is what it current looks like:

![Screenshot 2023-04-07 at 3 14 59 PM](https://user-images.githubusercontent.com/30158156/230930856-ba6a888d-cd9f-4aef-8841-dffd2dd2fe12.png)

Reply:
Made button take up whole width!